apps/crm/src/styles/globals.css

```diff
@@ -0,0 +1,24 @@
+:root {
+  --mailchimp-yellow: #ffe01b;
+  --mailchimp-brown: #241c15;
+}
+
+body {
+  font-family: Inter, -apple-system, "system-ui", "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell,
```

Comment:
**suggestion**: this can be removed

If you apply `ThemeProvider` from Macaw it automatically should apply the `Inter` font family to the body.